src/repos/FlyerRepo.ts

```diff
@@ -0,0 +1,229 @@
+import Filter from 'bad-words';
+import { ObjectId } from 'mongodb';
+import Fuse from 'fuse.js';
+import { Flyer, FlyerModel } from '../entities/Flyer';
+import {
+  DEFAULT_LIMIT,
+  MAX_NUM_DAYS_OF_TRENDING_ARTICLES,
+  FILTERED_WORDS,
+  DEFAULT_OFFSET,
+} from '../common/constants';
+import { OrganizationModel } from '../entities/Organization';
+
+const { IS_FILTER_ACTIVE } = process.env;
+
+function isFlyerFiltered(flyer: Flyer) {
+  if (IS_FILTER_ACTIVE === 'true') {
+    if (flyer.isFiltered) {
+      // If the body has been checked already in microservice
+      return true;
+    }
+    const filter = new Filter({ list: FILTERED_WORDS });
+    return filter.isProfane(flyer.title);
```

Comment:
i agree with @katesliang 's comments from ur last PR 
> why do you check if a flyer is profane twice? does microservice only check flyer description or smth?

Reply:
thanks shungo

Reply:
yeah i think the microservice for articles only checks the body, so i was planning on doing the same thing for flyers, but i realized that flyers actually don't have a body (it's just an image) so I'll get rid of this part

Reply:
nice